### ISSUE TITLE
Node v8 + daemon failure ("cwd" must be a string)

### ISSUE THREAD

**Issue by aldanor** [jhuckaby/Cronicle#37]
It looks like it may be affected by this bug: https://github.com/indexzero/daemon.node/issues/41

Basically, installing the latest release of Cronicle on Node v8.9.1, setting up and starting yields this error:

```sh
# /opt/cronicle/bin/control.sh start
/opt/cronicle/bin/control.sh start: Starting up Cronicle Daemon...
child_process.js:403
    throw new TypeError('"cwd" must be a string');
    ^

TypeError: "cwd" must be a string
    at normalizeSpawnArguments (child_process.js:403:11)
    at Object.exports.spawn (child_process.js:496:38)
    at Function.module.exports.daemon (/opt/cronicle/node_modules/daemon/index.js:50:31)
    at module.exports (/opt/cronicle/node_modules/daemon/index.js:25:20)
    at __construct.__init (/opt/cronicle/node_modules/pixl-server/server.js:123:21)
    at __construct.startup (/opt/cronicle/node_modules/pixl-server/server.js:161:8)
    at Object.<anonymous> (/opt/cronicle/lib/main.js:29:8)
    at Module._compile (module.js:635:30)
    at Object.Module._extensions..js (module.js:646:10)
    at Module.load (module.js:554:32)
/opt/cronicle/bin/control.sh start: Cronicle Daemon could not be started
```

**Comment by jhuckaby** [comment]
Thank you.  I'm looking into this now.

**Comment by jhuckaby** [comment]
Unfortunately there are a *lot* of problems with Node 8, which are preventing Cronicle from launching.  I am working on this, but each fix is uncovering 2 more issues, and many are in separate modules.

**Comment by jhuckaby** [comment]
Should be fixed in v0.7.4.

https://github.com/jhuckaby/Cronicle/releases/tag/v0.7.4

Thanks!

**Comment by aldanor** [comment]
Thanks a million!
